CloudOnMobile/CoreWrapper/CoreWrapperSwift.swift

```diff
@@ -15,3 +15,14 @@ import Foundation
 func initSharedCore() {
     set_shared_core_callbacks()
 }
+
+func listFiles() -> Data? {
```

Comment:
We should also add documentation for internal properties to provide easy understanding for other developers.